jekyll/_cci2/server/v4.2/installation/phase-1-prerequisites.adoc

```diff
@@ -129,7 +129,7 @@ For high availability, and to avoid potential outages, you should provision subn
 | Kubernetes Version
 
 | 4.2.x
-| 1.22 - 1.24
+| 1.22 - 1.28
```

Comment:
the rest, including 4.1.x has 1.29 as its max, so i assume for 4.2.x, this would also be 1.29? 🤔 

```suggestion
| 1.22 - 1.29
```

Reply:
i didnt formally check  4.2 on 1.29 as 4.2 will be EOL once 4.6 is released. 1.28 is available on EKS till end of nov this year so more than enough time to migrate to 4.3
there are significant changes between 4.2 and 4.3 with the addition of machine provisioner and related services so I dont think its fair to say 1.29 is supported but also not worth testing given 4.2 is about to EOL

Reply:
ah, gotcha no worries.
I was coming back from an uninformed background;
I merely noticed all other changes in this PR was  -> 1.29 except 4.2 here.